data/dbconn.py

```diff
@@ -1116,10 +1150,29 @@ async def get_user_problems(self, handles):
     def get_unsolved_problem(self, solved, total, handles, rating):
         fset = []
         for x in total:
-            if x[2] not in [name[2] for name in solved] and not self.is_an_author(x[0], handles) and x[4] == rating:
+            if x[2] not in [name[2] for name in solved]  and x[4] == rating:
```

Comment:
Why remove is_an_author() ?

Reply:
the bot was unable to find problems with a given rating, by removing the above part issue was resolved